freeCodeCamp/freecodecamp-dark-vscode-theme
# Update status bar nav color to fCC style guide

Checklist:


Closes #8 

## How to test 

- `git checkout ISSUE-8-bottom-nav-color` in forked repo
- Run f5 per contributing docs
- See bottom nav match color of side nav

before: 

- using default vs code color 

![Screen Shot 2021-09-22 at 12 11 59](https://user-images.githubusercontent.com/5950956/134390218-bd3fdf00-2c38-42e0-9838-4452800e1657.png)


after: 

- using fCC brand color background 

![Screen Shot 2021-09-22 at 12 11 32](https://user-images.githubusercontent.com/5950956/134390229-ef24b9d7-fc11-47d6-ba68-6012bc54d152.png)
